soft_delete_sample.py

```diff
@@ -114,49 +76,48 @@ def deleted_vault_recovery(self):
         """
         # create vaults enabling the soft delete feature on each
         vault_to_recover = self.create_vault()
-        self._enable_soft_delete_for_vault(vault_to_recover)
         vault_to_purge = self.create_vault()
-        self._enable_soft_delete_for_vault(vault_to_purge)
-
         print('created vaults {} and {}'.format(vault_to_recover.name, vault_to_purge.name))
 
         # delete the vaults
         self.keyvault_mgmt_client.vaults.delete(self.config.group_name, vault_to_recover.name)
-        self._wait_on_delete_completed(None, 'vault', vault_to_recover.name)
         print('Deleted vault: {}'.format(vault_to_recover.name))
-
         self.keyvault_mgmt_client.vaults.delete(self.config.group_name, vault_to_purge.name)
-        self._wait_on_delete_completed(None, 'vault', vault_to_purge.name)
         print('Deleted vault: {}'.format(vault_to_purge.name))
 
         # list the deleted vaults
         deleted_vaults = self.keyvault_mgmt_client.vaults.list_deleted()
-        print('Deleted Vaults: \n{}'.format(self._serialize(deleted_vaults)))
+        for deleted_vault in deleted_vaults:
```

Comment:
We should probably still print "Deleted vaults:" here and later in the sample to explain what the list of vaults is for:
```suggestion
        print("Deleted vaults:")
        for deleted_vault in deleted_vaults:
```